src/app/services/user.service.ts

```diff
@@ -21,48 +22,16 @@ export class LoginUser extends BasicUser {
   password_hash;
 }
 
-// export class OldUser {
-//   id: number;
-//   login_name: string;
-//   display_name: string;
-//   password_hash: string;
-//   user_creation: string;
-//   time_played: number;
-//   profile_picture: string;
-//   last_figure: string;
-//   is_connected: boolean;
-//   is_dev: boolean;
-
-//   constructor(login: string, display: string, password: string) {
-//     this.login_name = login;
-//     this.display_name = display;
-//     this.password_hash = hash.MD5(password);
-//   }
-// }
-
-// export class OldLoginUser {
-//   id: number;
-//   login_name: string;
-//   display_name: string;
-//   user_creation: string;
-//   time_played: number;
-//   profile_picture: string;
-//   last_figure: string;
-//   is_connected: boolean;
-//   is_dev: boolean;
-// }
-// export class OldForeignUser extends OldLoginUser { }
-
 @Injectable({
   providedIn: 'root',
 })
 export class UserService {
   activeUser: BehaviorSubject<LoginUser>;
   profileUser: BasicUser;
 
-  private userEndpoint = environment.endpoint + 'user';
+  private endpoint = environment.endpoint + 'user';
```

Comment:
I would not rename it to only endpoint. This makes the added ` + 'user'` harder to notice.